PHPCI/Helper/BaseCommandExecutor.php

```diff
@@ -57,10 +55,9 @@
      * @param bool        $quiet
      * @param bool        $verbose
      */
-    public function __construct(BuildLogger $logger, $rootDir, &$quiet = false, &$verbose = false)
+    public function __construct(BuildLogger $logger, $rootDir, &$verbose = false)
```

Comment:
@mavimo, I'm pretty sure this is a very bad practice :-) but it was actually used on purpose.